flow/src/main/java/com/sequenceiq/flow/service/flowlog/FlowLogDBService.java

```diff
@@ -94,10 +98,29 @@ public String getSerializedString(Object object) {
         } catch (Exception e) {
             LOGGER.debug("Somehow can not serialize object to string, try another method..", e);
             objectAsString = JsonUtil.writeValueAsStringSilent(object);
+            if (objectAsString == null) {
+                LOGGER.debug("Cannot serialize with JsonUtils, try with gson");
+                objectAsString = getGsonWithExceptionSerializer().toJson(object);
+            }
+        }
+        if (objectAsString != null) {
+            LOGGER.debug("Serialized string: {}", objectAsString);
```

Comment:
Are you sure if we should log this? This can be a large string, maybe with sensitive information

Reply:
You are right, I changed to a generic message